packages/optimizer/lib/transformers/SeparateKeyframes.js

```diff
@@ -45,9 +42,28 @@ class SeparateKeyframes {
   constructor(config) {
     this.log_ = config.log.tag('SeparateKeyframes');
     this.minify = config.minify !== false;
+    this.enabled = !!config.separateKeyframes;
+    if (
+      this.enabled &&
+      (!isDependencyInstalled('postcss') ||
+        !isDependencyInstalled('postcss-safe-parser') ||
+        !isDependencyInstalled('cssnano-simple'))
+    ) {
+      this.enabled = false;
+      this.log_.debug(
```

Comment:
You were using `log_.warn` before, is there a reason to use `debug` here instead?

Reply:
fixed